### ISSUE TITLE
opsu!-android stutters during launch and during gameplay

### ISSUE THREAD

**Issue by chong601** [fluddokt/opsu#9]
opsu!-android stutters after loaded (the song not playing smoothly and after 10 secs, it smooths out) and occasionally comes back during random periods of time

Phone affected: Asus Zenfone 4 (A4500CG)
Android version: 4.4.2
Version affected: 0.7.0a

Possibly due to Intel unable to convert the bytecode to Intel-compatible code fast enough sometimes o.o

**Comment by fluddokt** [comment]
Yeah, I probably won't be able to fix this. Though I may still tinker around with the music player in the future.

**Comment by chong601** [comment]
0.8.0a kinda fixed the problem... song no longer stutters during startup
but it does stutter during replay export after the gameplay ends and sometimes (although rare) during gameplay where every hitobject slows down.
It seems that Intel processors can't handle the sheer tankyness of the opsu-android LOL
